Sources/Ferno/Application+Ferno.swift

```diff
@@ -99,98 +99,93 @@ extension Application {
 extension Application.Ferno {
     
     /// Deletes everything
-    public func delete(_ path: String..., on req: Request) throws -> EventLoopFuture<Bool> {
-        try self.delete(path, on: req)
+    public func delete(_ path: String...) async throws -> Bool {
+        try await self.delete(path)
     }
 
     /// Creates child
-    public func create<T: Content>(_ path: String..., body: T, on req: Request) throws -> EventLoopFuture<FernoChild> {
-        try self.create(path, body: body, on: req)
+    public func create<T: Content>(_ path: String..., body: T) async throws -> FernoChild {
+        try await self.create(path, body: body)
     }
 
     /// Overwrites everything at that location with the data
-    public func overwrite<T: Content>(_ path: String..., body: T, on req: Request) throws -> EventLoopFuture<T> {
-        try self.overwrite(path, body: body, on: req)
+    public func overwrite<T: Content>(_ path: String..., body: T) async throws -> T {
+        try await self.overwrite(path, body: body)
     }
 
     /// Updates location, but omitted values won't get replaced
-    public func update<T: Content>(_ path: String..., body: T, on req: Request) throws -> EventLoopFuture<T> {
```

Comment:
Ah, I missed that. Actually, it's worth keeping both APIs. Changing the API this way would be a breaking change for those who may use my fork. Since the PR is made for the ﻿vapor-community repo it's more like a minor issue than major, but I don't provide support for my fork anymore, so people should migrate when it's merged and the migration would be less painful if NIO API is available. 🤔

Reply:
I think that causing breaking changes is acceptable when a project hasn't reached it's 1.0 release yet. After 1.0, this would require a major version bump.

Reply:
In "beta" projects `_.x._` is usually used for major changes and `_._.x` for minor, but my PR is not merged at all, so technically this PR doesn't even remove anything related to Vapor 4, plus migration to Vapor 4 is kinda breaking change itself so `_.x._` will change anyway, so I think it's acceptable too 👌